src/content/docs/en/guides/client-side-scripts.mdx

```diff
@@ -73,7 +73,7 @@ To avoid bundling the script, you can use the `is:inline` directive.
 ```
 
 :::note
-Adding `type="module"` or any attribute other than `src` to a `<script>` tag will disable Astro's default bundling behavior, treating the tag as if it had an `is:inline` directive.
+Adding `type="module"` or any attribute other than `src` to a `<script>` tag will disable Astro's default bundling behavior, treating the tag as if it had an `is:inline` directive. The same will be true when the script is written in a JSX expression.
```

Comment:
```suggestion
Astro's default bundling behavior will be disabled in some situations. In particular, adding `type="module"` or any attribute other than `src` to a `<script>` tag will cause Astro to treat the tag as if it had an `is:inline` directive. The same will be true when the script is written in a JSX expression.
```
What do you think about something like this, @MoustaphaDev ?

Reply:
Looks good!